Testreglar/1413/Nett/nett-1.4.13b.json

```diff
@@ -0,0 +1,186 @@
+{
+	"id": "nett-1.4.13b",
+	"type": "web",
+	"kravTilSamsvar": "<p>Når innhold kommer til syne og så forsvinner ved at man flytter tastaturfokus til innholdet og deretter flytter fokus vekk, gjelder alle de følgende punktene:</p>\r\n<ul>\r\n<li>Det er en mekanisme for å avvise eller lukke det ekstra innholdet, uten å flytte tastaturfokuset og\r\n<ul>\r\n<li>Med mindre det ekstra innholdet formidler en inndatafeil eller ikke forstyrrer eller erstatter annet innhold</li>\r\n</ul>\r\n</li>\r\n<li>Det ekstra innholdet forblir synlig helt til\r\n<ul>\r\n<li>brukeren flytter fokus eller</li>\r\n<li>brukeren avviser innholdet eller</li>\r\n<li>informasjonen ikke lenger er gyldig</li>\r\n</ul>\r\n</li>\r\n</ul>",
+	"kolonner": [
+		{
+			"title": "2.2"
+		},
+		{
+			"title": "3.2"
+		},
+		{
+			"title": "3.3"
+		},
+		{
+			"title": "3.4"
+		},
+		{
+			"title": "3.5"
+		},
+		{
+			"title": "3.6"
+		},
+		{
+			"title": "3.7"
+		}
+	],
+	"side": "2.1",
+	"element": "3.1",
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Hvilken side tester du?",
+			"ht": "<p>Oppgi URL eller side-ID</p>",
+			"type": "tekst",
+			"kilde": [],
+			"label": "URL/Side-ID:",
+			"datalist": "Sideutvalg",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Finnes det innhold som kommer til syne ved tastaturfokus på testsiden?",
+			"ht": "<p>Tab gjennom elementer med tastaturet for å identifisere ekstra innhold som viser ved tastaturfokus.</p>\n<p>Dette er innhold der ekstra innhold kommer til syne og så forsvinner, når innholdet får tastaturfokus og deretter flytter tastaturfokus vekk.</p>\n<p>Du skal bare teste elementer som får tastaturfokus.</p>\n<p><strong>Merk:</strong> Innhold som kan betjenes med muspekeren, skal også kunne betjenes ved kun med tastaturet, men dette testes i krav 2.1.1 Tastatur.</p>",
```

Comment:
1. Skriv om setning? Veldig mange "innhold": "Dette er innhold der ekstra innhold kommer til syne og så forsvinner, når innholdet får tastaturfokus og deretter flytter tastaturfokus vekk." Forslag til to første setninger: " Tab gjennom elementer med tastaturet for å identifisere ekstra innhold som vises ved tastaturfokus. Dette er ekstra innhold som kommer til syne når elementet får tastaturfokus, og deretter forsvinner når tastaturfokus flyttes vekk"
2. Skriv om: "skal også kunne betjenes ved kun med..." forslag: "skal også kunne betjenes med tastaturet"